scripts/test/Muon/elemental_analysis_2/EA_auto_tab/ea_auto_tab_model_test.py

```diff
@@ -0,0 +1,239 @@
+import unittest
+import unittest.mock as mock
+from mantid.simpleapi import CreateEmptyTableWorkspace , DeleteWorkspace
+from Muon.GUI.ElementalAnalysis2.auto_widget.ea_auto_tab_model import EAAutoTabModel
+from mantid.api import mtd
+
+
+class EAAutoTabModelTest(unittest.TestCase):
+
+    def setUp(self):
+        self.model = EAAutoTabModel(None)
+
+    @staticmethod
+    def delete_if_present(workspace):
+        if workspace in mtd:
+            DeleteWorkspace(workspace)
+
+    def test_update_match_table_when_table_has_more_than_3_rows(self):
+        correct_table_entries = ["9999", "Detector 1", "Ag , Au , Fe"]
+        #Create source tables
```

Comment:
I'm getting a few errors in Pycharm because there isn't a space between # and the comment in several places in this file. Might be worth putting those spaces in?